app.rb

```diff
@@ -25,15 +27,20 @@ def initialize
   end
 
   def list_all_books
-    puts 'books'
+    @books.each { |book| puts "Title: \"#{book.to_hash.label['title']}\", Author: #{book.to_hash['author']}" }
   end
 
   def list_all_labels
-    puts 'labels'
+    @labels.each { |label| puts "Title: \"#{label.to_hash.label['title']}\", Author: #{label.to_hash['author']}" }
   end
```

Comment:
This method does not fetch data from the json files instead try using

`@books = file_read('data/books.json')` -- this will fetch data from the json file books

   ` puts 'No books found' if @books.empty?` -- return message if no books found
    
    `@books.each do |book| `--- loops to pring each book
    
      `puts "Publish date: #{book['publish_date']}"`
      
    
    use `#{book['attribute']}` like shown in the example above instead of `#{book.to_hash.label['attribute']}